### PR TITLE
feat: doc for albumentation crafter

### PR DESCRIPTION
**Pull request by rutujasurve94**
Doc for `albumentation` crafter

### CODE REVIEW

File: crafters/image/AlbumentationsCrafter/README.md [hunk 1]
```diff
@@ -4,6 +4,40 @@ This crafter provides access to any of the transforms from the [Albumentations](
...
+    ```
+
+3. Run with Jina CLI
+   ```bash
+    jina pod --uses docker://jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7 --port-out 55556
+    ```
+
+4. Conventional local usage with `uses` argument
+    ```bash
+    jina pod --uses hub/example/imagecrafter.yml --port-in 55555 --port-out 55556
```
Comment (nan-wang) [inline crafters/image/AlbumentationsCrafter/README.md:30]: It's not clear what is `hub/example/imagecrafter.yml`.

File: crafters/image/AlbumentationsCrafter/README.md [hunk 2]
```diff
@@ -4,6 +4,40 @@ This crafter provides access to any of the transforms from the [Albumentations](
...
+    ```
+
+5. Docker command
+
+   Specify the image name along with the version tag. The snippet below uses Jina version `1.0.7`
+
+   ```bash
+    docker pull jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7
```
Comment (nan-wang) [inline crafters/image/AlbumentationsCrafter/README.md:38]: I won't consider this is one way of using Pod images.

```suggestion
5. Docker command

   Specify the image name along with the version tag. The snippet below uses Jina version `1.0.7`

   ```bash
    docker pull jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7
```

File: crafters/image/AlbumentationsCrafter/tests/test_albumentationscrafter.py [hunk 3]
```diff
@@ -1,3 +1,6 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
```
Comment (nan-wang) [inline crafters/image/AlbumentationsCrafter/tests/test_albumentationscrafter.py:1]:
```suggestion
__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
```

File: crafters/image/AlbumentationsCrafter/__init__.py [hunk 4]
```diff
@@ -1,3 +1,6 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
```
Comment (nan-wang) [inline crafters/image/AlbumentationsCrafter/__init__.py:1]:
```suggestion
__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
```

File: crafters/image/AlbumentationsCrafter/README.md [hunk 5]
```diff
@@ -4,6 +4,40 @@ This crafter provides access to any of the transforms from the [Albumentations](
 
 > Note that Albumentation's transform are meant to be random, which is usually not what you want in indexing/search context. The crafter automatically adds `always_apply=True` to all transforms, but you should watch out for other sources of "randomness" (see AlbumentationsCrafter documentation for more detail)
 
+## Usage
+
+Users can use Pod images in several ways:
+
+1. Run with Docker (`docker run`)
+   ```bash
+    docker run jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7 --port-in 55555 --port-out 55556
```
Comment (nan-wang) [inline crafters/image/AlbumentationsCrafter/README.md:13]: When running directly with docker, one needs to either bind the container port or to use `--network host`.
```suggestion
    docker run --rm -p 55555:55555 -p 55556:55556 jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7 --port-in 55555 --port-out 55556
```

File: crafters/image/AlbumentationsCrafter/README.md [hunk 6]
```diff
@@ -4,6 +4,40 @@ This crafter provides access to any of the transforms from the [Albumentations](
...
+2. Run with Flow API
+   ```python
+    from jina.flow import Flow
+    f = (Flow()
+        .add(name='my_crafter', uses='docker://jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7', port_in=55555, port_out=55556))
+    ```
+
+3. Run with Jina CLI
+   ```bash
+    jina pod --uses docker://jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7 --port-out 55556
```
Comment (nan-wang) [inline crafters/image/AlbumentationsCrafter/README.md:25]: I guess `--port-in` is missing here.
```suggestion
    jina pod --uses docker://jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7 --port-out 55556
```

File: crafters/image/AlbumentationsCrafter/README.md (hunk 2 above)
Comment (nan-wang) [reply crafters/image/AlbumentationsCrafter/README.md:38]: Please remove this part.

**Review by nan-wang — CHANGES_REQUESTED** [review]

File: crafters/image/AlbumentationsCrafter/README.md (hunk 5 above)
Comment (rutujasurve94) [reply crafters/image/AlbumentationsCrafter/README.md]: yes sure, added the suggestion

File: crafters/image/AlbumentationsCrafter/README.md (hunk 6 above)
Comment (rutujasurve94) [reply crafters/image/AlbumentationsCrafter/README.md]: added `--port-in`

File: crafters/image/AlbumentationsCrafter/README.md (hunk 1 above)
Comment (rutujasurve94) [reply crafters/image/AlbumentationsCrafter/README.md]: this is the config yml file for image crafter, renamed it

File: crafters/image/AlbumentationsCrafter/README.md [hunk 7]
```diff
@@ -4,6 +4,34 @@ This crafter provides access to any of the transforms from the [Albumentations](
...
+3. Run with Jina CLI
+   ```bash
+    jina pod --uses docker://jinahub/pod.crafter.albumentationscrafter:0.0.3-1.0.7 --port-in=55555 --port-out 55556
+    ```
+
+4. Conventional local usage with `uses` argument
+    ```bash
+    jina pod --uses hub/example/image_crafter_config.yml
+   
+   .yml --port-in 55555 --port-out 55556
```
Comment (bwanglzu) [inline crafters/image/AlbumentationsCrafter/README.md:32]: unexpected line break?

**Review by bwanglzu — CHANGES_REQUESTED** [review]

File: crafters/image/AlbumentationsCrafter/README.md (hunk 7 above)
Comment (rutujasurve94) [reply crafters/image/AlbumentationsCrafter/README.md]: sorry, corrected this.

**Review by bwanglzu — DISMISSED** [review]
LGTM!

**Comment by rutujasurve94** [comment]
@bwanglzu @nan-wang thanks a lot for the review, we can unblock and merge this if it looks good.

**Review by deepankarm — CHANGES_REQUESTED** [review]
Same comments as https://github.com/jina-ai/jina-hub/pull/4962

**Comment by rutujasurve94** [comment]
Thanks for the review @deepankarm, I've incorporated the changes

**Review by deepankarm — APPROVED** [review]

**Pull request merged**